internal/service/datalakepipeline/resource_data_lake_pipeline_test.go

```diff
@@ -142,3 +151,62 @@ func configBasic(orgID, projectName, clusterName, pipelineName string) string {
 		}
 	`, orgID, projectName, clusterName, pipelineName)
 }
+
+func configBasicWithPluralDS(orgID, projectName, firstClusterName, secondClusterName, firstPipelineName, secondPipelineName string) string {
```

Comment:
function name is confusing as it adds not only plural ds but a cluster resource and a singular ds.

can't it be the same configBasic function creating all resources needed?

Reply:
I'd avoid that because configBasic is also called for the migration test, and it wouldn't be basic anymore because it actually creates two clusters and two data lakes in order to test the plural part of it.

Would you suggest a different name perhaps?

Reply:
@lantoli I am going to go ahead with this so I can get unblocked. Sorry about that, we can always change it in a follow-up PR.

Reply:
ok, that's fine